src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -167,8 +173,24 @@ public SnapshotSummaryModel retrieveSnapshotSummary(UUID id) {
      * @return a SnapshotModel = API output-friendly representation of the Snapshot
      */
     public SnapshotModel retrieveAvailableSnapshotModel(UUID id) {
+        return retrieveAvailableSnapshotModel(id, getDefaultIncludes());
+    }
+
+    /**
+     * Convenience wrapper around fetching an existing Snapshot object and converting it to a Model object.
+     * Unlike the Snapshot object, the Model object includes a reference to the associated cloud project.
+     *
+     * Note that this method will only return a snapshot if it is NOT exclusively locked.
+     * It is intended for user-facing calls (e.g. from RepositoryApiController), not internal calls that may require
+     * an exclusively locked snapshot to be returned (e.g. snapshot deletion).
+     * @param id in UUID formant
+     * @param include a list of what information to include
+     * @return a SnapshotModel = API output-friendly representation of the Snapshot
```

Comment:
We don't have a lot of javadoc so I don't know if we have a shared style on how to write javadocs. But I don't usually include the type of the return object (since the javadoc will generate it).
```suggestion
     * @return an API output-friendly representation of the Snapshot
```